app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class AttachmentsController < ApplicationController
+  include Attachinary::ViewHelpers
+
+  skip_after_action :verify_authorized
+
+  def signature
+    render json: attachment_signature.to_json
+  end
+
+  def attachment_signature
```

Comment:
What do you think about making this method `private` since you don't need it in the public interface of the class?